doc/ref_model/chapters/chapter08.md

```diff
@@ -1,4 +1,320 @@
+
+The generic model for an enterprise cloud has to be "hybrid" with the special cases of purely private or public clouds as subsets of the generic hybrid cloud deployment model. In a hybrid cloud deployment, at least two or more distinct cloud infrastructures are inter-connected together.
+
+Cloud deployments can be created using a variety of technologies  (e.g., OpenStack, Kubernetes) and commercial technologies (e.g., VMware, AWS, Azure, etc.). A multi-cloud deployment can consist of the use of more than one technology.
+
+A generic Telco cloud is an hybrid multi-cloud. A better designation would be a federation of clouds - a federated cloud:
+   - a collection of cooperating, interoperable autonomous component clouds
+   -  the component clouds perform their local operations (internal requests) while also participating in the federation and responding to other component clouds (external requests)
+        - the component clouds are autonomous in terms of, for example, execution autonomy
+        - execution autonomy is the ability of a component cloud to decide the order in which internal and external requests are performed
+   - the component clouds are loosely coupled where no no changes are required to participate in a federation
+        - also, a federation controller does not impose changes to the component cloud except for running some central component(s) of the federated system (for example, a broker agent – executes as a workload)
+   - the component clouds are likely to differ in, for example, infrastructure resources and their cloud platform software
+   - workloads may be distributed on single or multiple clouds, where the clouds may be colocated or geographically distributed
+   - component clouds only surface NBIs (Please note that VMware deployed in a private and  a public cloud can be treated as a single cloud instance)
+
+<a name="8.2.1"></a>
+### 8.2.1 Characteristics of a Federated Cloud
+In this section we will further explore the characteristics of the federated cloud, architecture and architecture building blocks that constitute the  federated cloud. For example, Figure 8-1 shows a Telco Cloud that consists of 4 sub-clouds: Private on prem, Cloud Vendor provided on prem, Private outsourced (Commercial Cloud Provider such as an Hyperscalar Cloud Provider (HCP), and Public outsourced (see diagram below). Such an implementation of a Telco Cloud allows for mix'n'match of price points, flexibility in market positioning and time to market, capacity with the objective of attaining near "unlimited" capacity, scaling within a sub-cloud or through bursting across sub-clouds, access to "local" capacity near user base, and access to specialised services.
+
+<p align="center"><img src="../figures/RM-Ch08-HMC-Image-1.png" alt="Example Hybrid Multi-Cloud Component Cloud"><br><b>Figure 8-1: Example Hybrid Multi-Cloud Component Cloud.</b></p>
```

Comment:
"Cloud Vendor" is starred in Cloud B without explanation.

Reply:
@CsatariGergely Thanks. Updated the diagram

BTW the pptx files were already uploaded